samples/word/50-document/manage-custom-xml-part.yaml

```diff
@@ -0,0 +1,222 @@
+order: 7
+id: word-document-manage-custom-xml-part
+name: Manage a CustomXmlPart
+description: 'This sample shows how to add, query, edit, and delete a custom XML part in a document.'
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#add-custom-xml-part").click(() => tryCatch(addCustomXmlPart));
+        $("#query").click(() => tryCatch(query));
+        $("#insert-attribute").click(() => tryCatch(insertAttribute));
+        $("#insert-element").click(() => tryCatch(insertElement));
+        $("#delete-custom-xml-part").click(() => tryCatch(deleteCustomXmlPart));
+
+        async function addCustomXmlPart() {
+          // Add a custom XML part.
+          await Word.run(async (context) => {
+            // You must have the xmlns attribute to populate the CustomXmlPart.namespaceUri property.
+            const originalXml =
+              "<Reviewers><Reviewer>Juan</Reviewer><Reviewer>Hong</Reviewer><Reviewer>Sally</Reviewer></Reviewers>";
+            const customXmlPart = context.document.customXmlParts.add(originalXml);
+            customXmlPart.load("id");
+            const xmlBlob = customXmlPart.getXml();
```

Comment:
seems this line can put after line 22 and we will be able to remove one extra context.sync()